src/Compilers/Core/Portable/DiagnosticAnalyzer/DiagnosticStartAnalysisScope.cs

```diff
@@ -672,18 +759,26 @@ internal struct AnalyzerActions
         private ImmutableArray<CompilationStartAnalyzerAction> _compilationStartActions;
         private ImmutableArray<CompilationAnalyzerAction> _compilationEndActions;
         private ImmutableArray<CompilationAnalyzerAction> _compilationActions;
+
         private ImmutableArray<SyntaxTreeAnalyzerAction> _syntaxTreeActions;
         private ImmutableArray<AdditionalFileAnalyzerAction> _additionalFileActions;
+
+        private ImmutableArray<SemanticModelStartAnalyzerAction> _semanticModelStartActions;
+        private ImmutableArray<SemanticModelAnalyzerAction> _semanticModelEndActions;
         private ImmutableArray<SemanticModelAnalyzerAction> _semanticModelActions;
```

Comment:
i broke these into groups to make the groupings very clear, and to keep the pattern of start/end/action consistent.